ddtrace/internal/_encoding.pyx

```diff
@@ -569,8 +569,9 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
         has_span_type = <bint> (span.span_type is not None)
         has_meta = <bint> (len(span._meta) > 0 or dd_origin is not NULL)
         has_metrics = <bint> (len(span._metrics) > 0)
+        has_parent_id = <bint> (span.parent_id is not None)
```

Comment:
should we omit if it is `0` as well?

```suggestion
        has_parent_id = <bint> (span.parent_id is not None or span.parent_id == 0)
```

not sure how much it would actually matter, so feel free to ignore/resolve this comment

Reply:
yeah we only use `None` in the library so probably worth avoiding the additional check

Reply:
yup, that is the right call 👍🏻